lib/bgs/dependents.rb

```diff
@@ -88,6 +88,7 @@ def report_divorce
           divorce_state: formatted_info['divorce_state'],
           divorce_city: formatted_info['divorce_city'],
           divorce_country: formatted_info['divorce_country'],
+          end_date: formatted_info['end_date'],
```

Comment:
should the expected hash in the spec include an end_date now?

Reply:
Yep, I'm glad you mentioned that bc it made me realize we needed another attribute. Added 👍